.github/workflows/feature.yml

```diff
@@ -21,3 +21,8 @@ jobs:
         npm run lint
       env:
         CI: true
+    - name: verify docs were not updated
+      run: |
+        git checkout develop
+        git checkout -
+        git diff develop HEAD --exit-code --quiet -- docs
```

Comment:
Is it okay "-- docs"?

Reply:
yeah, that's to indicate to check the diff only in the docs folder. Since is a feature branch, that folder shouldn't have changed.

Reply:
Okay, I was curious if should be --docs or -- docs, but it that works as it's desired, it's okay for me (y)